blockchains/ethereum/info/info.json

```diff
@@ -1,22 +1,7 @@
-{
```

Comment:
"name":"YYFi.Protocol",
   "website":"https://yyfiprotocol.com/",
   "source_code":"https://github.com/yyfiprotocol",
   "white_paper":"https://yyfiprotocol.com/yyfiwhitepaper.pdf",
   "short_description":"yyfi.protocol is the largest global Digital Currency Based on Crypto Network.",
   "explorer":"https://etherscan.io/tx/0x3976c83011029d9f9a0f6710c39b2fecb4d4aeb60099ca3e8e862392194c6bc2",